share/revng/test/tests/abi/abi.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+/*
+ * This file is distributed under the MIT License. See LICENSE.md for details.
+ */
+
+#define UNLIMITED 65536
+
+// This file contains a dirty trick. Ideally we'd want to automagically import
+// these values from the corresponding abi `yaml` files contained in revng, but
+// for now manually copying the values here will do.
+
+#if defined(ABI_SystemV_x86_64)
+#define ABIDEF
```

Comment:
What is this for?

Reply:
Yeah, empty definitions are actually useless. I'll backport the updated `abi.h` here after you review it on the other PR.

Reply:
These are gone.